src/ir/attr_functor.h

```diff
@@ -105,6 +106,7 @@ class AttrFunctor<R(const ObjectRef& n, Args...)> {
   virtual R VisitAttr_(const tir::CastNode* op, Args... args) ATTR_FUNCTOR_DEFAULT;
   virtual R VisitAttr_(const tir::CallNode* op, Args... args) ATTR_FUNCTOR_DEFAULT;
   virtual R VisitAttr_(const tir::SelectNode* op, Args... args) ATTR_FUNCTOR_DEFAULT;
+  virtual R VisitAttr_(const SEScopeNode* op, Args... args) ATTR_FUNCTOR_DEFAULT;
```

Comment:
Is it possible to make SEScopeNode a subclass Attrs, in this case we do not need special handling here

Reply:
Yes, looks like that will work.

In general it seems anything POD-like should be an AttrNode<T>.

Reply:
Would be nice to leave as a TODO if we do not plan to handle directly here.

Reply:
Just gotten back to this -- so we agree adding equality & hash on Target is the way forward? We'll have to do that if we want SEScope to be 'just an attrs', right?

Reply:
I personally agree that equality hashing on Target is the way to go